modules/uhttp/filters_test.go

```diff
@@ -67,10 +67,13 @@ func TestTracingFilterWithLogs(t *testing.T) {
 		response := testServeHTTP(chain, host)
 		assert.Contains(t, response.Body.String(), "filters ok")
 		assert.True(t, len(buf.Lines()) > 0)
+		var spanFinished bool
 		for _, line := range buf.Lines() {
 			assert.Contains(t, line, "traceID")
 			assert.Contains(t, line, "spanID")
+			spanFinished = strings.Contains(line, "Span finished")
```

Comment:
It's minor redundancy in a test Alex :P However the reason is we can't rely on the buf lines order, that would make the test fragile if it's not exactly in the last line and an extra log was added somewhere else.